src/components/odata/data-access.vue

```diff
@@ -25,15 +25,19 @@ except according to the terms contained in the LICENSE file.
   </span>
 </template>
 
-<script>
-export default {
-  name: 'OdataDataAccess',
-  props: {
-    analyzeDisabled: Boolean,
-    analyzeDisabledMessage: String
-  },
-  emits: ['analyze']
-};
+<script setup>
```

Comment:
This component was very simple, so I also took the opportunity to convert it to a Composition API component.